common/ast/src/parser/statement.rs

```diff
@@ -717,3 +820,27 @@ pub fn auth_type(i: Input) -> IResult<AuthType> {
         value(AuthType::JWT, rule! { JWT }),
     ))(i)
 }
+
+// parse: (k = v ...)* into a map
+pub fn options(i: Input) -> IResult<BTreeMap<String, String>> {
+    let ident_with_format = alt((
+        map(rule! { FORMAT }, |_| "FORMAT".to_string()),
```

Comment:
An alternative way: remove `FORMAT` from the reserved ident list.

Reply:
I think it's better to put FORMAT as reserved keyword.

Thus in the case of `select * from t format CSV`,  we can't take format as the alias of table t.